### DISCUSSION TITLE
how to avoid keyclu being triggered by BetterTouchTool tip-tap/switch between chrome tabs shortcut

### DISCUSSION THREAD

**Discussion by avichou** [Anze/KeyCluCask#57] [Q&A] [answered]
i set tip tap with trackpad to trigger "control+tab" in order to go between chrome tabs. however since the last 2 updates, this seems to trigger keyclu and "awake" it 

any help would be appreciated :) thanks 

<img width="639" alt="image" src="https://github.com/Anze/KeyCluCask/assets/86660671/da10b1bf-a6f1-4762-b127-11a41f51a052">

https://github.com/Anze/KeyCluCask/assets/86660671/0960e765-60fb-472b-8e7c-9c2cfc09f501

**Comment by Anze** [comment, marked as answer]
Hey [avichou](https://github.com/avichou),
Seems like you've hidden the app icon, and that turned on App to respond to a triple press of the control key. I was considering getting rid of this feature, and I'll likely do it in the next release. Unfortunately the only way to avoid issue for now is to unhide icon.

**Reply by avichou** [reply]
thank you. the workaround does indeed work . i'll wait for the next release to re-toggle hiding :)
